stan/truncated_delay_model.stan

```diff
@@ -0,0 +1,19 @@
+data {
+  int<lower = 0> n;
+  array[n] real<lower = 0> onset_to_hosp;
+  array[n] real<lower = 0> time_since_onset;
+}
+
+parameters {
+  real meanlog;
+  real<lower = 0> sdlog;
+}
+
+model {
+  meanlog ~ normal(0, 10);
+  sdlog ~ normal(0, 10) T[0, ];
+
+  for (i in 1:n) {
+    onset_to_hosp[i] ~ lognormal(meanlog, sdlog) T[0, time_since_onset[i]];
```

Comment:
Hmm is this equivalent to dividing by the CDF? I think it isn't but I would be hard pressed to give a ref to that effect. Is there are model problems thought this is where I would expect them to be coming from.

Reply:
Reading this again would certainly seem to imply that this works: https://mc-stan.org/docs/stan-users-guide/truncation-censoring.html#truncated-data.section

Reply:
When I tried both they seemed to give the same results.